pkg/services/apiserver/config.go

```diff
@@ -41,6 +41,9 @@ func applyGrafanaConfig(cfg *setting.Cfg, features featuremgmt.FeatureToggles, o
 	o.AggregatorOptions.ProxyClientCertFile = apiserverCfg.Key("proxy_client_cert_file").MustString("")
 	o.AggregatorOptions.ProxyClientKeyFile = apiserverCfg.Key("proxy_client_key_file").MustString("")
 
+	o.AggregatorOptions.APIServiceCABundleFile = apiserverCfg.Key("apiservice_ca_bundle_file").MustString("")
```

Comment:
Does this mean that the same CA bundle file will be used for every apiservice?

Reply:
Good question. I was also wondering about this.

Since, this is for internally managed grafana.app API groups, I was thinking yes.

How is the cloud-apps-platform APIService going to be added to the server? If that process is external, it can set its own CABundle. I raise a similar comment in the Google doc for direct API Server access.